extlinks/links/admin.py

```diff
@@ -1,10 +1,21 @@
 from django.contrib import admin
+from django.contrib.contenttypes.admin import GenericTabularInline, GenericStackedInline
 
 from .models import URLPattern, LinkSearchTotal, LinkEvent
 
 
+class LinkEventURLPatternAdminInline(GenericTabularInline):
+    model = LinkEvent
+    show_change_link = True
+    exclude = ['username', 'user_id', 'on_user_list', 'url']
+
+
 class URLPatternAdmin(admin.ModelAdmin):
     list_display = ("url",)
+    list_select_related = ["collection"]
```

Comment:
Since we are no longer using `collections`, let's exclude it and add a comment as to why we're excluding it (old db schema).